manifests/w/Wingware/WingIDE101/9.0.0.4/Wingware.WingIDE101.locale.en-US.yaml

```
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: Wingware.WingIDE101
+PackageVersion: 9.0.0.4
+PackageLocale: en-US
+Publisher: Wingware
+PackageName: WingIDE 101

```

```suggestion
PackageName: Wing 101
```
This should match Control Panel